tests/OAuth2SummitApiTest.php

```diff
@@ -1818,4 +1819,132 @@ public function testGenerateQREncKey()
         $content = $response->getContent();
         $this->assertResponseStatus(201);
     }
+
+    public function testAddLeadReportSettings(){
+
+        $params = [
+            'id' => self::$summit->getId(),
+        ];
+
+        $allowed_columns = [
+            'scan_date',
+            'attendee_first_name',
+            'attendee_company',
+            'attendee_extra_questions' => [
+                [
+                    'id'   => 392,
+                    'name' => 'QUESTION1'
+                ],
+            ],
+            'extra_questions' => [
```

Comment:
this test is wrong
at summit level
extra_questions => ['*'] bc u dont have an specific sponsor